src/components/periodic-report/periodic-report.resolver.ts

```diff
@@ -61,10 +61,15 @@ export class PeriodicReportResolver {
     description: 'Create periodic reports for projects and engagements',
   })
   async syncAllReports(@LoggedInSession() session: Session) {
```

Comment:
Does it matter if this is async?

Reply:
Looks like it doesn't matter. https://stackoverflow.com/questions/45594596/async-function-without-await-in-javascript

Reply:
Doesn't matter